app/views/posts/index.html.erb

```diff
@@ -31,5 +31,5 @@
     <% end %>
   </ul>
 
-  <%= 'This user has not posted anything yet!' if @user.posts.empty? %>
+  <%= 'This user has not scribbled anything yet!' if @user.posts.empty? %>
```

Comment:
- [ ] [OPTIONAL] I tried scribbling a new scribble :laughing:  and the scribble threw an error, kindly fix this scribbling app :roll_eyes: 
![Screenshot from 2023-06-29 12-51-52](https://github.com/preciousbetine/microverse-rails-blog/assets/66097371/dfc4da77-127c-4db8-b2b4-a05e8c94c619)

Reply:
@VICTOR-CHUKWU lol thanks. Will look into it.

Reply:
The error is raised only when you try to create a post without signing in. I think that's what the next part of the project deals with.